### DISCUSSION TITLE
Hide "Topics in this Section"

### DISCUSSION THREAD

**Discussion by JorgeR1986** [google/docsy#707] [Q&A]
Hello!

Is there a way to hide the "Topics in this Section" at the bottom of the Index Pages?

Thank you!

**Comment by emckean** [comment]
Hi! Could you share an example of what you're talking about?

**Comment by aimeeu** [comment]
@JorgeR1986  

To hide a page or section from the left navigation menu, set toc_hide: true in the front matter.

To hide a page from the section summary on a docs section landing page, set hide_summary: true in the front matter. If you want to hide a page from both the TOC menu and the section summary list, you need to set both toc_hide and hide_summary to true in the front matter.

https://www.docsy.dev/docs/adding-content/navigation/#section-menu
